pkg/storaged/anaconda.jsx

```diff
@@ -47,13 +47,17 @@ export function remember_passphrase(block, passphrase) {
     if (!window.isSecureContext)
         return;
 
-    try {
-        const passphrases = JSON.parse(window.sessionStorage.getItem("cockpit_passphrases")) || { };
-        passphrases[device_name(block)] = passphrase;
-        window.sessionStorage.setItem("cockpit_passphrases", JSON.stringify(passphrases));
-    } catch {
-        console.warn("Can't record passphrases");
-    }
+    // Let's put this code back once we know that we really need
```

Comment:
We still need it because of Anaconda still re-scanning and teaning down devices. https://issues.redhat.com/browse/INSTALLER-3898